monitor.py

```diff
@@ -35,8 +35,24 @@ def fetch_release_date(release):
                 if p_ele:
                     if "Created:" in p_ele:
                         start_date = p_ele.split(" ")[1]+" "+p_ele.split(" ")[2]
-                        break
-            return start_date
+                        return start_date
+            form = soup.find("form", {"method": "GET"})
```

Comment:
Can we move this before checking entire elements in the block. i.e before https://github.com/ocp-power-automation/ci-monitoring-automation/blob/main/monitor.py#L31